addons/repair/functions/fnc_addRepairActions.sqf

```diff
@@ -72,6 +64,28 @@ private _turretPaths = ((fullCrew [_vehicle, "gunner", true]) + (fullCrew [_vehi
         _action = [_name, _text, _icon, _statement, _condition, {}, [_hitpoint], _position, 2] call EFUNC(interact_menu,createAction);
         [_type, 0, [], _action] call EFUNC(interact_menu,addActionToClass);
 
+        // Create a wheel interaction
+        private _root = format ["Wheel_%1_%2", _forEachIndex, _hitpoint];
+        private _action = [_root, localize "str_xbox_wheel", ["","#FFFFFF"], {}, {true}, {}, [_hitpoint], _position, 2, nil, LINKFUNC(modifySelectionInteraction)] call EFUNC(interact_menu,createAction);
```

Comment:
`str_xbox_wheel` translation is bit weird, I think it's for XBOX Steering Wheel controller and does not make sense in all langauges.

Reply:
Ah yeah good catch